src/main/java/ui/Friday.java

```diff
@@ -0,0 +1,62 @@
+package ui;
+
+import commands.Command;
+
+import common.BotMessage;
+import common.TaskList;
+
+import exceptions.FridayException;
+
+import parser.Parser;
+
+import storage.Storage;
+
+public class Friday {
+    private final BotMessage header = new BotMessage("""
```

Comment:
Perhaps add a descriptor as to what header this is for? The variable 'ending' too